mods/example/mod.yaml

```diff
@@ -6,16 +6,24 @@ Packages:
 	.
 	$example: example
 	./mods/common: common
-	example|bits
```

Comment:
Why does it still work if you remove this?

Reply:
Probably because everything was prefixed with `bits/`.

Reply:
I prefer proper paths instead of one huge virtual folder.

Reply:
Yes, this is another good practice that more mods should be following.

Reply:
Removing this line however breaks the asset browser.